wiki/Ranking_Criteria/Ranking_Criteria_Council/en.md

```diff
@@ -0,0 +1,65 @@
+
+### osu! members
+
+- [Charles445](https://osu.ppy.sh/users/85000) (American mappers)
+- *(vacant)* (American players)
+- [Cherry Blossom](https://osu.ppy.sh/users/1156742) (European mappers)
+- [Doomsday](https://osu.ppy.sh/users/18983) (European players)
+- *(vacant)* (Asian mappers)
+- [Frostmourne](https://osu.ppy.sh/users/199669) (Asian players)
+- [Garven](https://osu.ppy.sh/users/244216) (QAT)
+- [Mao](https://osu.ppy.sh/users/2204515) (QAT)
+
+### osu!taiko members
+
+- [OzzyOzrock](https://osu.ppy.sh/users/465153) (American mappers)
+- [Shyguy](https://osu.ppy.sh/users/178038) (American players)
+- [Nwolf](https://osu.ppy.sh/users/1910766) (European mappers)
+- *(vacant)* (European players)
+- [qoot8123](https://osu.ppy.sh/users/766371) (Asian mappers)
+- [bbj0920](https://osu.ppy.sh/users/87546) (Asian players)
+- [MMzz](https://osu.ppy.sh/users/128993) (QAT)
+- [DakeDekaane](https://osu.ppy.sh/users/1425253) (QAT)
+
+### osu!catch members
+
+- [ZiRox](https://osu.ppy.sh/users/200768) (American mappers)
+- [Zak](https://osu.ppy.sh/users/1375955) (American players)
+- [JBHyperion](https://osu.ppy.sh/users/4879508) (European mappers)
+- *(vacant)* (European players)
+- [Spectator](https://osu.ppy.sh/users/702598) (Asian mappers)
+- [He Ang Erika](https://osu.ppy.sh/users/2451381) (Asian players)
+- [Deif](https://osu.ppy.sh/users/318565) (QAT)
+- [Kurokami](https://osu.ppy.sh/users/260933) (QAT)
+
+### osu!mania members
+
+- [Fullerene-](https://osu.ppy.sh/users/2531335) (American mappers)
+- [Halogen-](https://osu.ppy.sh/users/169992) (American players)
+- [-Kamikaze-](https://osu.ppy.sh/users/2124783) (European mappers)
+- [Hayabusa](https://osu.ppy.sh/users/3104108) (European players)
+- [iJinjin](https://osu.ppy.sh/users/2141612) (Asian mappers)
+- [Cryolien](https://osu.ppy.sh/users/1626983) (Asian players)
+- [Starry-](https://osu.ppy.sh/users/2166199) (QAT)
+- [Blocko](https://osu.ppy.sh/users/4075092) (QAT)
+
+The Ranking Criteria Council was ultimately unsuccessful. Only members involved in osu!catch promoted new proposals, while the other game modes did not make progress until the Ranking Criteira Council was disbanded and replaced with the United Beat-knights of Ranking Criteria.
+
+## United Beat-Knights of Ranking Criteria
+
+The United Beat-Knights of Ranking Criteira (*UBKRC*) was a new group with similar goals to the Ranking Criteria Council, however it was handled with less strictness. The group chose an intentionally stupid name to appear more friendly than its predecessor. The group was managed by [Okoratu](https://osu.ppy.sh/users/1623405) and [pishifat](https://osu.ppy.sh/users/3178418) until completion.
+
+Within the UBKRC, teams of experts were chosen to handle each area of the general Ranking Criteria and game-mode specific sections. Users could join if they showed an active interest in any group.
```

Comment:
https://docs.google.com/spreadsheets/d/1PBF3XOLIWOScTxmSzzF1-fqjIaDhrfX3rPxcw6IAP5M/edit#gid=0 should have most or all of said people listed who worked on UBKRC. Feels like it should be listed if the RC council is.